faraday/proxy.py

```diff
@@ -452,13 +463,13 @@ def proxy():
         # present, create port queue for it and append data to that queue
         try:
             data["data"]
-        except:
+
+        except KeyError:
             logger.error("Error: No 'data' key in dictionary")
             return json.dumps(
                 {"error": "Error: No 'data' key in dictionary"}), 400
         else:
```

Comment:
This `else` is unnecessary because the function will return if the `except` block is executed.